code/game/objects/items/robot/cyborg_gripper.dm

```diff
@@ -201,6 +213,19 @@ as part of their multi-layered protections to try and eliminate the chance of a
 		/obj/item/toy/plushie
 	)
 
+////////////////////////////////
+// MARK:	MINING GRIPPER
+////////////////////////////////
+// For mining borgs. Mostly for self-application of goliath armour.
+/obj/item/gripper/mining
+	name = "mining gripper"
+	desc = "A grasping tool for cyborgs. This version is ruggedized for better endurance in the harsh enviroments of Lavaland."
```

Comment:
You should probably suggest the items that this can hold (or imply that through its applications) in either its desc or an examine().

Reply:
Altered.